### ISSUE TITLE
Capture area is pretty off from selection area

### ISSUE THREAD

**Issue by notNSANE** [graemephi/imgur-uploader#6]
I'm using the latest Vivaldi snapshot but the issue should happen on stable too since it's quite recent.

This is an example when I tried to frame the 5 cat pictures on top left

![image](https://user-images.githubusercontent.com/7162375/131207551-4804494c-8217-4f0a-baac-e545e5b53828.png)

**Comment by graemephi** [comment]
I can't repro this, on stable or on 2392.3  (the August 19 snapshot), on neither windows or macos. It looks they changed something with their own capture tool in that snapshot:

```
[Capture] Selection Area Misplaced if UI Zoom not 100% (VB-81763)
```

But their bug tracker is closed so it's hard to find out more. Anyway, I'm getting the right capture in both versions regardless of zoom. This extension has to correct for zoom itself, so I can see something like this happening if the browser behaviour changes. One thing I can't really test, or fix if it is the issue, is a windows machine with DPI-aware scaling.

In the past, canary versions of chrome have broken the APIs this extension makes use of so, while it is good to know behaviour might have changed ahead of time, my tendency here is to wait until it hits a stable release.

**Comment by ksdavidc** [comment]
This has been bugging me for some time. However, I noticed by accident that under a different account it was working properly. After some troubleshooting I think I may have a clue to the source of the problem. 
If you set the DEFAULT page zoom (from chrome settings chrome://settings/?search=size) to 100% it works, otherwise it offsets. The SITE page zoom (from the three-dot popup menu far right on the url bar, whatever it's called) has no effect. 

The desired behavior would be to make it work not based on the default zoom but the page zoom.

I hope that helps fix it, cause the extension is good, but this bughas prevented me from using it. I have found one near alternative that almost works for me, but this is more convenient to use...

**Comment by graemephi** [comment]
Yup, easy to reproduce on my end. I also took the time to update the extension to manifest V3, which turned out to remove the need for the zoom correction on our side anyway (with manifest V3, we have to do everything with the active tab's DOM; before, we had to translate between the zoomed page DOM and the unzoomed extension DOM. I didn't realise that at the time I wrote it through, and implicitly assumed the extension DOM was always at 100%).

So this should be fixed with version 2.4, which is now pending review on the chrome web store.

**Comment by ksdavidc** [comment]
nice work sir! Looking forward to using it in style! Thank you.

**Comment by 097115** [comment]
Capture doesn't seem to work for me at all?

I right click on the page, [select `Capture area`](https://i.imgur.com/FPhTz14.png), and then just nothing happens. What am I doing wrong? :)

Brave Browser Version 1.36.122 Chromium: 99.0.4844.88 (Official Build) (x86_64)
imgur Uploader 2.4.0
